### PR TITLE
Newsletter 13: Bevy Engine

### PR DESCRIPTION
**Pull request by cart**
Part of #232

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ Table of contents:
 
 - [Game Updates](#game-updates)
 - [Learning Material Updates](#learning-material-updates)
-- [Library & Tooling Updates](#library-tooling-updates)
+- [Library & Tooling Updates](#library--tooling-updates)
+  - [Bevy Engine](#bevy-engine)
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:33]: To keep it from being super long, we don't usually add the individual sections to the TOC.
```suggestion
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Looks good for the most part! Only nitpicks:

* To keep it from being super long, we don't usually add the individual sections to the TOC
* Generally I think it's preferred to write the updates in third person ('they' rather than 'we'), just to keep a consistent tone throughout the newsletter

File: content/posts/newsletter-013/index.md (hunk 1 above)
Comment (cart) [reply content/posts/newsletter-013/index.md:33]: oops i didn't manually add this / this wasn't my intention. looks like my markdown vscode plugin manages table of contents automatically:

image

Sorry about that! I should have looked at the diff more carefully.

Comment (17cupsofcoffee) [reply content/posts/newsletter-013/index.md:33]: Someone else had the same thing happen in their PR, it's no problem :)

File: content/posts/newsletter-013/index.md [hunk 2]
```diff
@@ -66,6 +66,67 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+- [Editor-Ready UI](https://github.com/bevyengine/bevy/issues/254)
+- [Physically Based Rendering (PBR)](https://github.com/bevyengine/bevy/issues/179)
+- [Scenes](https://github.com/bevyengine/bevy/issues/255)
+
+_Discussions:
+[/r/rust](https://www.reddit.com/r/rust/comments/i7bcwu/introducing_bevy_a_refreshingly_simple_datadriven/),
+[hacker news](https://news.ycombinator.com/item?id=24123283),
+[twitter announcement](https://twitter.com/cart_cart/status/1292903435155599361),
+[amethyst forum]((https://community.amethyst.rs/t/bevy-engine-addressing-the-elephant-in-the-room))_
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:128]:
```suggestion
[amethyst forum](https://community.amethyst.rs/t/bevy-engine-addressing-the-elephant-in-the-room)_
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good now, thanks for making the changes! Just one issue with one of the links, but happy to approve in advance.

**Review by ozkriff — APPROVED** [review]
:rocket: (made a few style and fmt tweaks)

**Comment by ozkriff** [comment]
Merged, thanks for the PR! Good luck with this epic endeavor!

**Pull request merged**